src/raft_server_properties.c

```
@@ -288,3 +288,11 @@ raft_node_id_t raft_get_transfer_leader(raft_server_t *me)
     return me->node_transferring_leader_to;
 }
 
+void raft_get_server_stats(raft_server_t *me, raft_server_stats_t *stats)
+{
+    if (stats == NULL) {
+        return;
+    }
```

```suggestion
```

I don't think null check is good here. If user is calling this function with a null pointer, it seems like a bug to me and we should let it crash. Currently, we silently ignore the error.

---

done